src/icon-manifest/supportedExtensions.ts

```diff
@@ -262,6 +262,7 @@ export const extensions: IFileCollection = {
     { icon: 'reactjs', extensions: [], languages: [languages.javascriptreact], format: FileFormat.svg },
     { icon: 'reacttemplate', extensions: ['rt'], format: FileFormat.svg },
     { icon: 'reactts', extensions: [], languages: [languages.typescriptreact], format: FileFormat.svg },
+    { icon: 'reason', extensions: ['re', 'rei'], languages: [languages.reason], format: FileFormat.png },
```

Comment:
Remove the `extensions` references. Those are provided by the extension itself (https://github.com/freebroccolo/vscode-reasonml/blob/master/package.json#L181).